content/pages/disability-benefits/index.md

```diff
@@ -9,6 +9,7 @@ template: level2-index
 widgets:
   - root: react-applicationStatus
     timeout: 20
+    buildtype: staging
```

Comment:
So, I'm not sure this is a great way of doing this, since it's adding yet another way we can gate features. Open to other suggestions.

Reply:
As an alternative, can we comment out the widgets section here entirely until we're ready? Maybe add it to a launch checklist or something.

Reply:
We could, but you wouldn't be able to test out the widgets in staging, which can be confusing during final content passes before launch. But I don't feel super strongly either way.